### PR TITLE
Fix#14495: error with P type and comment suffix

### PR DESCRIPTION
**Pull request by realitix**
Fixed issue #14495

### CODE REVIEW

**Comment by Shnoulle** [comment]
See mantis comment : must fix it in model ( import, api, plugin etc … must use same rules)

**Comment by realitix** [comment]
I updated the pull request with your indication. It works!

**Comment by Shnoulle** [comment]
@realitix OK, seems excellent, thank you.
Just a final question : did you try to fix https://github.com/LimeSurvey/LimeSurvey/blob/51f685f31172bee68802a0885600568377d89845/application/helpers/admin/activate_helper.php#L22 in this pull request ?

- if yes : i wait for the fix
- if no : i squash and merge and see what happen with a previous survey :)

File: application/models/Question.php [hunk 1]
```diff
@@ -134,6 +134,10 @@ public function rules()
             if ($oParentQuestion->other == "Y") {
                 $aRules[] = array('title', 'LSYii_CompareInsensitiveValidator', 'compareValue'=>'other', 'operator'=>'!=', 'message'=> sprintf(gT("'%s' can not be used if the 'Other' option for this question is activated."), "other"), 'except' => 'archiveimport');
             }
+            // #14495: comment suffix can't be used with P Question (collapse with table name in database)
+            if ($oParentQuestion->type == "P") {
+                $aRules[] = array('title', 'match', 'pattern'=>'/.+comment$/', 'not'=>true, 'message'=> gT("'comment' suffix can not be used with question of type P."));
```
Comment (Shnoulle) [inline application/models/Question.php]: Unsure of what to do with archiveimport ? I think we can't activate since some years : then we can't have LSA.

**Comment by realitix** [comment]
Helle @Shnoulle.
I didn't try to fix the `fixNumbering` function because with this pull request, the problem is fixed earlier.
So you can merge it.

Thanks

**Comment by Shnoulle** [comment]
OK, great :)

**Pull request merged**